spec/services/broadcasts/welcome_notification/introduction_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe Broadcasts::WelcomeNotification::Introduction, type: :service do
+  describe "::send" do
+    let(:receiving_user) { create(:user) }
+    let(:user) { create(:user) }
+    let(:mascot_account) { create(:user) }
+    let!(:welcome_broadcast) { create(:welcome_broadcast, :active) }
+
+    before do
+      allow(User).to receive(:mascot_account).and_return(mascot_account)
+      SiteConfig.staff_user_id = mascot_account.id
+    end
+
+    after do
+      SiteConfig.staff_user_id = 1
+    end
```

Comment:
something to note is that we'll need to do this setup across all welcome notifications